### ISSUE TITLE
S3 error setting up `s3_scratch` fixture

### ISSUE THREAD

**Issue by gjoseph92** [coiled/coiled-runtime#293]
`PermissionError: The unspecified location constraint is incompatible for the region specific endpoint this request was sent to.`

This happens during `s3.mkdirs`:

https://github.com/coiled/coiled-runtime/blob/5d13ab90ff0006efac70f7586fcac0038508d36f/conftest.py#L313-L320

**Comment by ntabris** [comment]
@ian-r-rose is this the semi-mysterious error you were trying to reproduce?

**Comment by ian-r-rose** [comment]
Yeah, possibly related to #240 . Nothing seemed to cause any problems then.

@gjoseph92 do you have a default region set in your aws config?

**Comment by gjoseph92** [comment]
I do, `us-east-2`

**Comment by ian-r-rose** [comment]
Weird. Try upgrading s3fs and boto?
